releasing-archway.md

```diff
@@ -0,0 +1,111 @@
+# Archway Protocol Release Versioning
+
+Reference: [Semantic Versioning](http://semver.org)
+
+Legend:
+
+- **X.Y.Z** refers to the version (git tag) of Archway Protocol that is released.
+- **Network Operator** refers to an entity running a node and/or a validator and/or a relayer.
+- **archway-1** refers to the chain id of archway protocl main network.
+- **constantine-1** refers to the chain-id of the archway protocol canary public test network.
+- **titus-1** refers to the the chain-id of the archway protocol development public test network.
```

Comment:
Should titus be for private development test network and constantine-1 a public one?

Reply:
@jlehtimaki I do not think so, dapp developers and community might need access to a public network to try out beta features early on

Reply:
But these "features" may be cut from release if they fail to scale or introduce security vulnerabilities or other issues;
Would this not be a poor developer experience if developers "test" features in their dApps only to see it cut?

Titus being considered unstable with expected state reset seems more like an Alpha network than a Beta network;

Reply:
Release branch is only cut when its "feature complete", the responsibility to test features for scale and security falls on the developer before signing of as "complete". Feature may still be depricated in subsequent release and is considered normal in any software development cycle